front/screens/Profile.tsx

```diff
@@ -2,13 +2,15 @@ import type { StackNavigationProp } from "@react-navigation/stack";
 import { filter, find } from "lodash";
 import type { FC } from "react";
 import * as React from "react";
+// eslint-disable-next-line @typescript-eslint/no-duplicate-imports
```

Comment:
Idem, est-ce que cette ligne est nécessaire ?

Reply:
C'est pour éviter d'avoir une erreur eslint ('react' import is duplicated.)

Reply:
Essaie de remplacer la ligne 4 par `import React from "react"` pour voir si ça fait disparaître l'erreur

Reply:
Non on ne peut pas, ça génère une autre erreur : No default export found in imported module "react".

Reply:
Bon alors gardons cet eslint-disable